src/dnddiceroller/Ordo_DiceTower.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package ordo_dicetower;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author zacharyzordo
+ */
+public class Ordo_DiceTower {
+
+    /**
+     * @param args the command line arguments
+     */
+    
+    public static void main(String[] args) {
+        boolean d6Pass = true;
+        boolean towerPass = true;
+	for(int i = 0; i< 1000; i++){
+            int d6 = test_oneD6();
+            if(d6 != -1){
+		System.out.println("Die Test Failed with Value: " + d6);
+                d6Pass = false;
+            }
+            int tower = test_diceTowerWithTwoD6();
+            if(tower != -1){
+                System.out.println("Tower Test Failed with Value: " + tower);
+                towerPass = false;
+            }   
+        }
+        if(d6Pass){
+            System.out.println("Die Test Passed");
+        }
+        if(towerPass){
```

Comment:
I like the addition of pass statements with the booleans. I was confused my first run of it with no output.